src/App.test.js

```diff
@@ -30,4 +30,11 @@ describe('App', () => {
       screen.getByTestId('error-message')
     ).toBeInTheDocument()
   })
+  it('renders an items table', async () => {
```

Comment:
Suggestion: Add a test for the sad path here, since you handled it! Ex: this test becomes "When there are items, renders an items table." and the new test is "When there are no items, does not render an items table."